cmd/cloud/deployment.go

```diff
@@ -347,6 +351,10 @@ func deploymentCreate(cmd *cobra.Command, _ []string, out io.Writer) error {
 		return errors.New("Invalid --high-availability value")
 	}
 
+	if organization.IsOrgHosted() && !(clusterType == standard || clusterType == dedicated) {
```

Comment:
minor nit: but we should have `cmd.SilenceUsage = true` at the end of all validations, instead of doing it before these validations.

The difference if we move that at the end of all validations would be along with the error message CLI would also log the usage of the command, i.e. the output of `astro deployment create --help` so that the user doesn't have. to explicitly run the help command after seeing just the error message

Reply:
oh yes!! Good call out. Will move that in the end